boards/arm/stm32h747i_disco/doc/index.rst

```diff
@@ -0,0 +1,209 @@
+- Ethernet compliant with IEEE802.3-2002
+- USB OTG HS
+- Stereo speaker outputs
+- ST-MEMS digital microphones
+- 256-Mbit SDRAM
+- 4 color user LEDs
+- 1 user and reset push-button
+- 4-direction joystick with selection button
+- Arduino Uno V3 connectors
+
+.. image:: img/stm32h747i_disco.jpg
+     :width: 2362px
+     :align: center
+     :height: 1763px
+     :alt: STM32H747I-DISCO
+
+More information about the board can be found at the `STM32H747I-DISCO website`_.
+More information about STM32H747XIH6 can be found here:
+
+- `STM32H747XI on www.st.com`_
+- `STM32H747xx reference manual`_
+- `STM32H747xx datasheet`_
+
+Supported Features
+==================
+
+The current Zephyr stm32h747i_disco board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on Zephyr porting.
+
+The default configuration per core can be found in the defconfig files:
+``boards/arm/stm32h747i_disco/stm32h747i_disco_defconfig_m7``
+``boards/arm/stm32h747i_disco/stm32h747i_disco_defconfig_m4``
+
+Pin Mapping
+===========
+
+STM32H747I Discovery kit has 9 GPIO controllers. These controllers are responsible for pin muxing,
+input/output, pull-up, etc.
+
+For mode details please refer to `STM32H747I-DISCO website`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- UART_1 TX/RX : PA9/PA10 (ST-Link Virtual Port Com)
+- UART_8 TX/RX : PJ8/PJ9 (Arduino Serial)
+- LD1 : PI12
+- LD2 : PI13
+- LD3 : PI14
+- LD4 : PI15
+- W-UP : PC13
+- J-CENTER : PK2
+- J-DOWN : PK3
+- J-LEFT : PK4
+- J-RIGHT : PK5
+- J-UP : PK6
+
+System Clock
+============
+
+The STM32H747I System Clock can be driven by an internal or external oscillator,
+as well as by the main PLL clock. By default, the CPU1 (Cortex-M7) System clock
+is driven by the PLL clock at 400MHz, and the CPU2 (Cortex-M4) System clock
+is driven at 200MHz. PLL clock is feed by a 25MHz high speed external clock.
+
+Serial Port
+===========
+
+The STM32H747I Discovery kit has up to 8 UARTs.
+Default configuration assigns USART1 and UART8 to the CPU1. The Zephyr console
+output is assigned to UART1 which connected to the onboard ST-LINK/V. Virtual
```

Comment:
Not sure what the right fix is here, but should this be ``ST-LINK/Virtual``

Reply:
Should be ST-LINK/V3.0. Txs